### ISSUE TITLE
MeasureTheory.jl as a tool to learn Measure Theory

### ISSUE THREAD

**Issue by arbenede** [cscherrer/MeasureTheory.jl#91]
This is more an idea/invitation than an issue. I have struggled for a long time to learn Measure Theory and never got much beyond the first few chapters of **many** MT books... I think that this package could be a vehicle to explain and teach the basics of measures to an audience interested in using it in their day to day work more than in learning its foundations. So I really hope that someone will write a tutorial with examples. For example, this package should allow to do regression on a mixture of discrete and continuous random variables, correct? That is actually a problem I worked on some time ago...

**Comment by mschauer** [comment]
Yeah, you can define such a [mixture](https://github.com/cscherrer/MeasureTheory.jl/blob/master/src/combinators/spikemixture) and use it in [inference](https://discourse.julialang.org/t/ann-zigzagboomerang-jl/57287/3?u=mschauer).

**Comment by cscherrer** [comment]
Thanks @arbenede , I think this is a great idea. Most of the references I know of are very theoretical, and I think there could be an opportunity for something with a more applied focus, and as you say lots of examples.

I wrote a post on Zulip today that you might find helpful as part of an introduction:
https://julialang.zulipchat.com/#narrow/stream/259730-measuretheory.2Ejl/topic/Base.20measures.2C.20representatives.2C.20etc
